presto-spi/src/main/java/io/prestosql/spi/connector/StandardWarningCode.java

```diff
@@ -22,6 +22,7 @@
     TOO_MANY_STAGES(0x0000_0001),
     REDUNDANT_ORDER_BY(0x0000_0002),
     DEPRECATED_FUNCTION(0x0000_0003),
+    REDIRECTED_TABLE(0x0000_0004),
```

Comment:
`TABLE_REDIRECTION` sounds better to me, since it makes the "redirection" the noun rather than the "table". Thoughts?

Reply:
Sounds better to me, too